__template/package.json

```
@@ -8,16 +8,16 @@
     "typecheck": "tsc"
   },
   "dependencies": {
-    "@remix-run/node": "*",
-    "@remix-run/react": "*",
-    "@remix-run/serve": "*",
+    "@remix-run/node": "~1.14.2",
```

What about using carets (`^`) instead, so we can still have v1.15 if we would release that?

```suggestion
    "@remix-run/node": "^1.14.2",
```

---

1.15 will introduce the deprecations which is what we're trying to avoid here

---

Just the deprecation warnings, no?
So everything would still work, no?

Or is that the thing we don't want to have in the examples: the deprecation warnings?

---

> Just the deprecation warnings, no?
> So everything would still work, no?

correct, everything will still work, but with deprecation warnings - we have a plan to update/ask contributors of examples to update, but that's not needed actively

> should we also update the stacks?
stacks can be updated around the time of 1.15 being released to opt into future flags

---

Gotcha! 👍